src/core/Akka.Remote.TestKit.Tests/RemoteConnectionSpecs.cs

```diff
@@ -62,13 +69,13 @@ public RemoteConnectionSpecs() : base(Config)
                 var address = RARP.For(Sys).Provider.DefaultAddress;
 
                 // have the client send a message to the server
-                await client.WriteAndFlushAsync(new Hello("test", address));
+                client.WriteAndFlushAsync(new Hello("test", address));
                 var hello = serverProbe.ExpectMsg<Hello>();
```

Comment:
That should be more than enough time - I suspected this test was hanging on the `await`-ed calls for whatever reason